tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1995,7 +1996,13 @@ std::unique_ptr<KernelThunk> IrEmitterUnnested::BuildKernelThunk(
                 << hlo_buffer_slice->gte_index.ToString();
 
         bindings_.BindHloToIrValue(*instr, value, index);
-      });
+      },
+      // Check temp buffer numbers only when the multiheap mode is off.
+      /*insist_single_temp_buffer=*/inst->parent()
```

Comment:
isn't this bool same as `insist_single_temp_buffer`?

Reply:
Yes. Note that this is the caller to generate the boolean flag though.